.env.example

```diff
@@ -1,8 +1,8 @@
 # required keeper wallet and provider url
 ETH_HDWALLET_MNEMONIC="test test test test test test test test test test test junk"
-PROVIDER_API_KEY_INFURA=
-PROVIDER_API_KEY_ALCHEMY=
+PROVIDER_URL_ALCHEMY=
```

Comment:
I removed `_API_KEY_{PROVIDER_NAME}` in favour of of the entire URL because of `JsonRpcProvider`. Infura/Alchemy providers don't support `getSigner(address)` 😭 which we need here because of the `onlyAssociatedContracts` modifier on `addressesPage`.

Reply:
I did see that, the example just missing `PROVIDER_URL_INFURA`  only has `PROVIDER_URL_ALCHEMY`.. no biggie of course